sql/core/src/test/scala/org/apache/spark/sql/DynamicPartitionPruningSuite.scala

```diff
@@ -1528,6 +1528,22 @@ abstract class DynamicPartitionPruningSuiteBase
       }
     }
   }
+
+  test("SPARK-37995: Fix DPP does not compile nested subquery in AQE") {
```

Comment:
Thank you, @ulysses-you . Could you revise the test case name and PR title a little bit?

Reply:
thank you @dongjoon-hyun . I have updated it, hope it is more readable than before.